integration/main_test.go

```diff
@@ -88,17 +87,6 @@ func ConnectDaemons() error {
 	if err != nil {
 		return fmt.Errorf("failed to create image service: %w", err)
 	}
-	// Since CRI grpc client doesn't have `WithBlock` specified, we
-	// need to check whether it is actually connected.
-	// TODO(#6069) Use grpc options to block on connect and remove for this list containers request.
-	_, err = runtimeService.ListContainers(&runtime.ContainerFilter{})
```

Comment:
the cri service grpc isn't ready even if the listener starts to accept connection. The CRI plugin will take time to recover the state from container and task services before ready. So I think we should keep it.

Reply:
Ahh, makes sense. Let me think about this, I'll draft for now

Reply:
Not sure if I can on mobile.. I'll draft later today then if not 🤣